doc/userguide/install.rst

```diff
@@ -170,18 +170,85 @@ Follow these steps from your Suricata directory::
 Binary packages
 ---------------
 
-Ubuntu
-^^^^^^
+Ubuntu from Personal Package Archives (PPA)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-For Ubuntu, the OISF maintains a PPA ``suricata-stable`` that always contains the latest stable release.
+For Ubuntu, OISF maintains a PPA ``suricata-stable`` that always contains the
+latest stable release.
 
-To use it::
+Setup to install the latest stable Suricata::
 
     sudo apt-get install software-properties-common
     sudo add-apt-repository ppa:oisf/suricata-stable
     sudo apt-get update
+
+Then, you can install the latest stable with::
+
     sudo apt-get install suricata
 
+If you want Suricata with built-in (enabled) debugging::
+
+    sudo apt-get install suricata-dbg
+
+After installing you can proceed to the :ref:`Basic setup`.
+
+`OISF launchpad: suricata-stable <https://launchpad.net/~oisf/+archive/suricata-stable>`_.
+
+Upgrading
+"""""""""
+
+To upgrade::
+
+    sudo apt-get update
+    sudo apt-get upgrade suricata
+
+Remove
+""""""
+
+To remove Suricata from your system::
+
+    sudo apt-get remove suricata
+
+Getting Beta or RC Releases
```

Comment:
I'd spell out `RC`, at least the first time.  Or simply use `Pre-release Versions`.